typhos/utils.py

```diff
@@ -1058,3 +1069,30 @@ def get_text(item):
 
         print(separator, file=file)
         return file.getvalue()
+
+
+def get_config_path():
+    """Get the typhos configuration path."""
+    if platform.system().lower() == 'windows':
+        path = os.getenv('APPDATA') or '~'
+    elif platform.system().lower() == 'darwin':
+        path = pathlib.Path('~') / 'Library' / 'Caches'
+    else:
+        path = os.getenv('XDG_CACHE_HOME') or '~/.cache'
+
+    # Allow TYPHOS_CONFIG_PATH to override the above operating system-specific
+    # entries:
+    path = os.environ.get('TYPHOS_CONFIG_PATH', path / 'typhos')
```

Comment:
This fails on linux because path is a string, unlike macos which is pre-converted to a `pathlib.Path`

Reply:
D'oh - I had rewritten this like 3 times, thinking I had gotten the most simple version. Will fix.